bionic/dagviz.py

```diff
@@ -14,6 +14,26 @@
 Image = import_optional_dependency('PIL.Image', purpose=module_purpose)
 
 
+class SvgImage(object):
```

Comment:
Since Bionic only supports Python 3 now, I _think_ you can get away with "new-style" classes:
`class SvgImage` (inheriting from `object` implicitly).